README.md

```diff
@@ -58,6 +58,11 @@ Note: we currently only support and test on python 3.7, use other versions at yo
 pytest -xsv tests/
 ```
 
+Note: `PYTHONPATH` must include the `timemachine` repo root. To run tests that use `openeye`, ensure that either `OE_LICENSE` or `OE_DIR` are set. For example, starting from a clean environment with the openeye license file in `~/.openeye`:
+```
+OE_DIR=~/.openeye PYTHONPATH=. pytest -xsv tests/
```

Comment:
I would combine this with the lines above where you run the tests, as if you don't have OE license setup a user would have the line above fail.

Reply:
Good call, thanks. Updated in db650ca.